controller/authController.ts

```diff
@@ -0,0 +1,25 @@
+import type {Request, Response, Application, response} from "express";
```

Comment:
As a team you may want to consider how you are structuring this application, a src folder for the source code and a tests folder for the test code would maybe make the build process easier